islandora_datastream_exporter.drush.inc

```diff
@@ -102,28 +115,21 @@ function islandora_datastream_export_create_batch() {
  * @param array $context
  *   The context of the Drupal batch.
  */
-function islandora_datastream_export_batch_operation($query, $dsid, $target, &$context) {
-  $sandbox = &$context['sandbox'];
+function islandora_datastream_export_batch_operation($type, $query, $dsid, $target, &$context) {
+  drupal_static_reset('islandora_get_tuque_connection');
   $export_objects = 10;
-  $qp = new IslandoraSolrQueryProcessor();
-  $qp->buildQuery($query);
-  if (!isset($sandbox['offset'])) {
-    $qp->executeQuery(FALSE);
-    $sandbox['total'] = $qp->islandoraSolrResult['response']['numFound'];
-    $sandbox['offset'] = 0;
+  if (isset($type['file'])) {
+    call_user_func_array('module_load_include', $type['file']);
   }
-  $context['message'] = t('Processing results @start to @end.', array(
-    '@start' => $sandbox['offset'],
-    '@end' => min($sandbox['offset'] + $export_objects, $sandbox['total']),
+  module_load_include('inc', 'islandora', 'includes/mimetype.utils');
+  module_load_include('inc', 'islandora', 'utilities');
```

Comment:
Should be `includes/utilities`

Reply:
Interesting that that worked before ... must get included somewhere up the line :/

Fixed it